### PR TITLE
media-libs/gexiv2: bump to 0.12.0

### PR DESCRIPTION
**Pull request by lovesegfault**
This also syncs the 9999 ebuild with the newest one.

Closes: https://bugs.gentoo.org/691774
Package-Manager: Portage-2.3.71, Repoman-2.3.17
Signed-off-by: Bernardo Meurer <bernardo@standard.ai>

### CODE REVIEW

**Comment by lovesegfault** [comment]
@leio Could you take a look at this?

**Comment by mgorny** [comment]
Would you be interested in becoming a proxied maintainer for this? Graphics team is not exactly healthy.

**Comment by lovesegfault** [comment]
@mgorny Yep, I'll maintain it :)

File: media-libs/gexiv2/gexiv2-0.12.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6} )
```
[the remark this answers is not in the document]
Comment (lovesegfault) [reply media-libs/gexiv2/gexiv2-0.12.0.ebuild]: Apparently not! Added :)

File: media-libs/gexiv2/gexiv2-0.12.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+if [[ ${PV} == 9999 ]]; then
+	EGIT_REPO_URI="https://gitlab.gnome.org/GNOME/gexiv2.git"
+	inherit git-r3
+else
+	SRC_URI="mirror://gnome/sources/${PN}/$(ver_cut 1-2)/${P}.tar.xz"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-fbsd"
+fi
+
+LICENSE="LGPL-2.1"
```
[the remark this answers is not in the document]
Comment (lovesegfault) [reply media-libs/gexiv2/gexiv2-0.12.0.ebuild]: Fixed :)

File: media-libs/gexiv2/gexiv2-0.12.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+	use vala && vala_src_prepare
+	default
+}
+
+src_configure() {
+	local emesonargs=(
+		$(meson_use introspection)
+		$(meson_use vala vapi)
+		$(meson_use gtk-doc gtk_doc)
+		# prevents installation of python modules (uses install_data from meson
```
[the remark this answers is not in the document]
Comment (lovesegfault) [reply media-libs/gexiv2/gexiv2-0.12.0.ebuild]: Fixed :)

File: media-libs/gexiv2/gexiv2-0.12.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+"
+RDEPEND="${PYTHON_DEPS}
+	>=dev-libs/glib-2.38.0:2
+	>=media-gfx/exiv2-0.21:=
+	introspection? ( >=dev-libs/gobject-introspection-1.54:= )
+"
+DEPEND="${RDEPEND}"
+
+src_prepare() {
+	xdg_environment_reset
```
[the remark this answers is not in the document]
Comment (lovesegfault) [reply media-libs/gexiv2/gexiv2-0.12.0.ebuild]: I highly doubt it, removed.

File: media-libs/gexiv2/gexiv2-0.12.0.ebuild [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+"
+RDEPEND="${PYTHON_DEPS}
+	>=dev-libs/glib-2.38.0:2
+	>=media-gfx/exiv2-0.21:=
+	introspection? ( >=dev-libs/gobject-introspection-1.54:= )
+"
+DEPEND="${RDEPEND}"
+
+src_prepare() {
+	use vala && vala_src_prepare
```
Comment (mgorny) [inline media-libs/gexiv2/gexiv2-0.12.0.ebuild]: By the way, that phase is a misnomer. You can safely move this call to `src_configure` (where it belongs) and get rid of redefining `src_prepare`.

Comment (lovesegfault) [reply media-libs/gexiv2/gexiv2-0.12.0.ebuild]: Fixed :)

**Pull request closed without merging**